pyproject.toml

```diff
@@ -18,11 +18,10 @@
 #
 # Release a new version:
 #
-#  - git tag VERSION
-#  - git push --tags
-#  - Remove untracked files/dirs: git clean -fdx
-#  - python -m build
-#  - twine upload dist/*
+#  - go to the GitHub release tab: https://github.com/python/pyperformance/releases
+#  - click "Draft a new release" and fill the contents
+#  - finally click the "Publish release" button! Done!
+#  - monitor the publish status: https://github.com/python/pyperformance/actions/workflows/publish.yml
```

Comment:
FWIW, it may be worth keeping the old steps as an alternative:

```suggestion
# Release a new version with GitHub (preferred):
#
#  - go to the GitHub release tab: https://github.com/python/pyperformance/releases
#  - click "Draft a new release" and fill the contents
#  - finally click the "Publish release" button! Done!
#  - monitor the publish status: https://github.com/python/pyperformance/actions/workflows/publish.yml
#
# Release a new version manually:
#
#  - git tag VERSION
#  - git push --tags
#  - Remove untracked files/dirs: git clean -fdx
#  - python -m build
#  - twine upload dist/*
```